integration-tests/src/test/java/oracle/kubernetes/operator/ITUsabilityOperatorHelmChart.java

```diff
@@ -596,4 +581,127 @@ public void testCreateWithDefaultTargetDomainInstall() throws Exception {
     }
     logger.info("SUCCESS - " + testMethodName);
   }
+  /**
+   * Create operator and verify its deployed successfully. Create domain1 and verify domain is
+   * started. Call helm upgrade to add domainnew to manage, verify both domains are managed by
+   * operator Call helm upgrade to remove first domain from operator target domains, verify it can't
+   * not be managed by operator anymore
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testAddRemoveDomainUpdateOperatorHC() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    logger.info("Creating Operator & waiting for the script to complete execution");
+    // create operator
+    Map<String, Object> operatorMap = TestUtils.createOperatorMap(number, true);
+    Operator operator = new Operator(operatorMap, RESTCertType.SELF_SIGNED);
+    operator.callHelmInstall();
+    Domain domain = null;
+    Domain domainnew = null;
+    boolean testCompletedSuccessfully = false;
+    logger.info("kubectl create namespace test" + (number + 1));
+    ExecCommand.exec("kubectl create namespace test" + (number + 1));
+    logger.info("create domain with UID : test" + number);
+    domain = TestUtils.createDomain(TestUtils.createDomainMap(number));
```

Comment:
This test method looks lengthy, can u move some of the code into private methods like createDomainAndVerify() or something like that..

Reply:
changed to add private methods